src/themes/default/components/core/blocks/Microcart/Product.vue

```diff
@@ -18,38 +18,36 @@
       <div class="flex flex-nowrap details">
         <div class="flex w-100 flex-wrap between-xs">
-          <div :class="{ 'w-100 pb10': !productsAreReconfigurable }">
-            <div>
-              <router-link
-                class="serif h4 name"
-                :to="productLink"
-                data-testid="productLink"
-                @click.native="$store.commit('ui/setMicrocart', false)"
-              >
-                {{ product.name | htmlDecode }}
-              </router-link>
-              <div class="h6 cl-bg-tertiary pt5 sku" data-testid="productSku">
-                {{ product.sku }}
-              </div>
-              <div class="h6 cl-bg-tertiary pt5 options" v-if="isOnline && !editMode && product.totals && product.totals.options">
-                <div v-for="opt in product.totals.options" :key="opt.label">
-                  <span class="opn">{{ opt.label }}: </span>
-                  <span class="opv" v-html="opt.value" />
-                </div>
-              </div>
-              <div class="h6 cl-bg-tertiary pt5 options" v-else-if="!editMode && product.options">
-                <div v-for="opt in product.options" :key="opt.label">
-                  <span class="opn">{{ opt.label }}: </span>
-                  <span class="opv" v-html="opt.value" />
-                </div>
-              </div>
-              <div class="h6 pt5 cl-error" v-if="product.errors && Object.keys(product.errors).length > 0">
-                {{ product.errors | formatProductMessages }}
+          <div>
+            <router-link
+              class="serif h4 name"
+              :to="productLink"
+              data-testid="productLink"
+              @click.native="$store.commit('ui/setMicrocart', false)"
+            >
+              {{ product.name | htmlDecode }}
+            </router-link>
+            <div class="h6 cl-bg-tertiary pt5 sku" data-testid="productSku">
+              {{ product.sku }}
+            </div>
+            <div class="h6 cl-bg-tertiary pt5 options" v-if="isOnline && !editMode && product.totals && product.totals.options">
+              <div v-for="opt in product.totals.options" :key="opt.label">
+                <span class="opn">{{ opt.label }}: </span>
+                <span class="opv" v-html="opt.value" />
               </div>
-              <div class="h6 pt5 cl-success" v-if="product.info && Object.keys(product.info).length > 0 && Object.keys(product.errors).length === 0">
-                {{ product.info | formatProductMessages }}
+            </div>
+            <div class="h6 cl-bg-tertiary pt5 options" v-else-if="!editMode && product.options">
+              <div v-for="opt in product.options" :key="opt.label">
+                <span class="opn">{{ opt.label }}: </span>
+                <span class="opv" v-html="opt.value" />
               </div>
             </div>
+            <div class="h6 pt5 cl-error" v-if="product.errors && Object.keys(product.errors).length > 0">
```

Comment:
same: v-if to computed eg. `hasProductErrors`

Reply:
done